opennlp-morfologik-addon/src/test/java/opennlp/morfologik/tagdict/POSTaggerFactoryTest.java

```diff
@@ -47,9 +48,16 @@
 
   private static ObjectStream<POSSample> createSampleStream()
       throws IOException {
-    MarkableFileInputStreamFactory sampleDataIn = new MarkableFileInputStreamFactory(
-        new File(POSTaggerFactory.class.getResource("/AnnotatedSentences.txt").getFile()));
 
+    MarkableFileInputStreamFactory sampleDataIn;
+    try {
+      sampleDataIn =
```

Comment:
Do we need these changes to build on windows? Can you test this once more? A few problems should be fixed by now in master.